docs/creating-studies.md

```diff
@@ -115,9 +132,12 @@ styling.
   they have a small number of members**, i.e. less than 10.
 
 **Recommended**
-  - You may want to select a SQL style guide as a reference.
+  - You may want to select a SQL style guide as a reference. Mozilla provides a
+  [SQL style guide](https://docs.telemetry.mozilla.org/concepts/sql_style.html),
+  which our sqlfluff config enforces.
   [Gitlab's data team](https://about.gitlab.com/handbook/business-technology/data-team/platform/sql-style-guide/)
-  has an example of this, though there are other choices.
+  has a style guide that is more centered around DBT, but also has some practices
+  you may want to consider adopting.
```

Comment:
Should we just adopt that uniqueness into our sqlfluff config, and then we could drop mention of this extra guide? Or maybe it's more vibes based practices, and not so much enforcable practices

Reply:
there's a lil of it in there? but i think it's vibes.